rfcs/Issue_Triage_Guide.md

```diff
@@ -6,34 +6,35 @@ While the [common issue triage guide for O3DE](https://github.com/o3de/community
 
 This guide covers how to triage GitHub issues for SIG-Simulation. Maintainers are encouraged to use and update this guide to ensure all contributors to SIG-Simulation understands how issues are handled and accepted by the SIG.
 
-##  Issue Triaging
+## Issue Triaging
 
 Triaging is the process used to handle intake of issues into the SIG-Simulation backlog. The process aims to ensure issues are both relevant to SIG-Simulation and contain sufficient information so that the community can take action.
 
 Process aims to ensure that:
+
 * Issues are appropriate for SIG-Simulation. Confirms that issues are actual issues, rather than requests for help or issues for another SIG.
 * Issues have clear information to enable SIG-Simulation to address the problem or request.
-* Issues are regularly maintained and updated until they are resolved.  
+* Issues are regularly maintained and updated until they are resolved.
 * Issue load is balanced across SIG maintainers when action is required.
 * All the SIG-Simulation community can participate.
 
-# Process
+## Process
 
 SIG-Simulation triages issues once a week on [Thursday](https://lists.o3de.org/g/o3de-calendar/calendar). Anyone is welcome to attend. Triage will be led by SIG chair, co-chair or maintainer (referred to below as *Triage Leader*)
 
 ## Triage Leader Guide
 
 1. Join the SIG-Simulation discord voice channel
-2. Announce yourself as the Triage Leader and wait a few minutes for others to join the call.
-3. Use the *Individual Issue Triage* guide below to process all new issues for SIG:
-   1. Review any open issues with [needs-sig](https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-sig) that may be for SIG Simulation
-        1. Remove `needs-sig` and add `sig/simulation`. These items will now show up when reviewing issues below.    
-   2. Process the main repository issues
+1. Announce yourself as the *Triage Leader* and wait a few minutes for others to join the call.
+1. Use the *Individual Issue Triage* guide below to process all new issues for SIG:
+   1. Review any open issues with [needs-sig](https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-sig) that may be for SIG-Simulation
+        1. Remove `needs-sig` and add `sig/simulation`. These items will now show up when reviewing issues below.
+   2. Process the main repository issues. As issues are triaged, please see if they can be categorized and added to the [O3DE Simulation Bugs board](https://github.com/orgs/o3de/projects/26/views/1?groupedBy%5BcolumnId%5D=25435410).
```

Comment:
Should the numbers all be replaced with 1s in the nested lists as well? I just tested it on a markdown tester site and it seems like nested lists behave the same as top level lists.

Reply:
> I know this is a pretty small change, but ideally it would be easier to review if the formatting was a separate commit.

Apologies this is my Achilles heal, I find it hard not to tidy and change at the same time 🙈 I'll try and keep them separate next time 👍

Reply:
> Should the numbers all be replaced with 1s in the nested lists as well? I just tested it on a markdown tester site and it seems like nested lists behave the same as top level lists.

No increasing numbers are required, it can all be 1.1.1. really.. I have an extension that did auto-update them and I missed it before I posted the PR sorry